### PR TITLE
docs: update documentation to match v2.0.0 development

### PR DESCRIPTION
**Pull request by lionel-nj**
relates to https://github.com/MobilityData/gtfs-validator/issues/904

**Summary:**

This PR updates the documentation of v2.0.0 release to match the development of v2.0.0 version of the validator.

**Expected behavior:** 

No code change.

Please make sure these boxes are checked before submitting your pull request - thanks!

- ~[ ] Run the unit tests with `gradle test` to make sure you didn't break anything~
- [x] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [x] Linked all relevant issues
- ~[ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)~

### CODE REVIEW

**Comment by barbeau** [comment]
This shouldn't be merged to master though, right? It should stay in its own branch to be tagged as a replacement for v2 release? This branch should also branch off of the v2 release tag if the intent is to fix the v2 release docs.

**Comment by lionel-nj** [comment]
> This shouldn't be merged to master though, right? 

No it should not. 

>  It should stay in its own branch to be tagged as a replacement for v2 release? This branch should also branch off of the v2 release tag if the intent is to fix the v2 release docs.

Yes, it would come as a replacement for the v2 release

**Review by isabelle-dr — CHANGES_REQUESTED** [review]
@lionel-nj 
I added comments on what needs to be changed:
1- the `output` parameter that needs to be changed for `output_base`
2- the feed_name can't have capital letters, that is why Omar couldn't run this version of the validator

Can we also update the USAGE.md file with `output_base`?
Thanks!

File: README.md [hunk 1]
```diff
@@ -9,29 +9,56 @@ This command-line tool written in Java that performs the following steps:
 1. Checks file integrity, numeric type parsing and ranges as well as string format according to the [GTFS Schedule specification](https://gtfs.mobilitydata.org/spec/gtfs-schedule#h.hc443y62gb8c)
 1. Performs GTFS business rule validation *(work-in-progress)*
 
-# Build the code
-We suggest using [IntelliJ](https://www.jetbrains.com/idea/download/) to [import](https://www.jetbrains.com/help/idea/import-project-or-module-wizard.html), build, and run this project.
+# Run the app via command line
+### Setup
+1. Install [Java 8 or higher](https://www.oracle.com/java/technologies/javase-downloads.html)
+1. Download [gtfs-validator-v2.0.0_cli.jar](https://github.com/MobilityData/gtfs-validator/releases/download/v2.0.0/gtfs-validator-v2.0.0_cli.jar)
```
Comment (isabelle-dr) [inline README.md:15]: Add "Prepare your input directory. It has to be a compressed .zip folder containing all the GTFS feed text files. 
To compress a folder correctly, select all text files and select "Compress". Do not compress a folder from the folder icon."

**Comment by lionel-nj** [comment]
Closing as solved.

**Pull request closed without merging**